### ISSUE TITLE
Injury related to extreme wrist pronation or supination

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9907]
**Preferred term label:**
Injury related to extreme wrist pronation or supination

**Synonyms**
Twisting

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Monteggia fractures

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

in our universe, we would regard injury as a disease entity rather than a phenotypic feature. Possibly this could be made into a Mondo term?

**Comment by MickeySegal** [comment]
The wording of our label was not good and has been changed to "Extreme wrist pronation or supination by history", which fits as an historical finding.
